presentations/2024-01-25_coffee-and-coding/index.qmd

```diff
@@ -0,0 +1,149 @@
+---
+title: "Coffee and Coding"
+subtitle: "{targets}"
```

Comment:
Maybe the title could be more descriptive like "Making my analytical workflow more reproducible with {targets}" or something?